### DISCUSSION TITLE
Any file format specification for CR2W files? (at least abstract)

### DISCUSSION THREAD

**Discussion by quenbyako** [WolvenKit/WolvenKit#484] [Q&A] [answered]
So, i digging a lot into game files and still don't understand, how does these (looks like compiled) files can be converted to something useful...

i understand, that file format is closed, but actually you somehow working with it in this project... And i don't understand how... i want to make export scripts which are converting this files to models, textures etc, but i don't know where to start, honestly

**Comment by PixelRick** [comment, marked as answer]
You can check out our format spec + diagram here:
https://miro.com/app/board/o9J_l_E-PzQ=/

**Reply by quenbyako** [reply]
@PixelRick requested access, looks like this board is private...

**Reply by PixelRick** [reply]
Ah I thought it'd be public since we don't pay.. I have added public visibility, you can retry with the same link.

**Reply by quenbyako** [reply]
@PixelRick thanks a lot! will research all your guys proceedings!

Btw, even i don't understand csharp at all, i really appreciate job of all maintainers, this project helps me understand how red engine works, thanks again, you rock 🤟

**Reply by PixelRick** [reply]
The format spec is not in a specific language so you should be able to understand it! I am going to add a sticky note with definitions for the type names.
